docs/source/portfolio_optimisation/theory_implied_correlation.rst

```diff
@@ -0,0 +1,164 @@
+
+ - Marcenko-Pastur distribution is fit to the eigenvalues of the correlation matrix and the maximum theoretical eigenvalue is calculated.
+
+ - This maximum theoretical eigenvalue is set as a threshold and all the eigenvalues above the threshold are shrinked.
+
+ - The de-noised correlation matrix is calculated back from the eigenvectors and the new eigenvalues.
+
+.. tip::
+
+    The algorithm for de-noising the correlation and the covariance matrix is implemented in the RiskEstimators class of the mlfinlab package. It is described in more detail `here <https://github.com/hudson-and-thames/research/blob/master/RiskEstimators/RiskEstimators.ipynb>`_.
+
+
+.. tip::
+
+    This algorithm is described in more detail in the work **Estimation of Theory-Implied Correlation Matrices** *by* Marcos Lopez de Prado `available here <https://papers.ssrn.com/abstract_id=3484152>`_.
+
+Correlation matrix distance
+###########################
+
+The similarity of the Empirical correlation matrix and the Theory-implied correlation matrix can be measured using the correlation matrix distance introduced by Herdin and Bonek `in this paper <https://publik.tuwien.ac.at/files/pub-et_8791.pdf>`_.
+
+The distance is calculated as:
+
+.. math::
+   d[\sum_{1},\sum_{2}] = 1 - \frac{tr(\sum_{1}\sum_{2})}{||\sum_{1}||_f||\sum_{2}||_f}
+
+Where :math:`\sum_{1},\sum_{2}` are the two correlation matrices and the :math:`||.||_f` is the Frobenius norm.
+
+From the work **Estimation of Theory-Implied Correlation Matrices**:
+"The distance :math:`\sum_{1},\sum_{2}` measures the orthogonality between the considered correlation matrices. It becomes zero if the correlation matrices are equal up to a scaling factor, and one if they differ to a maximum extent".
+
+Implementation
+##############
+
+.. automodule:: mlfinlab.portfolio_optimization.tic
+
+    .. autoclass:: TIC
+        :members:
+
+        .. automethod:: __init__
+
+Example Code
+############
+
+.. code-block::
+
+    import pandas as pd
+    from mlfinlab.portfolio_optimization.tic import TIC
+
+    # Reading data
+    tree_classification = pd.read_csv('TREE_FILE_PATH')
+    stock_returns = pd.read_csv('DATA_FILE_PATH', parse_dates=True, index_col='Date')
+
+    # Calculating the empirical correlation matrix
+    corr_matrix = stock_returns.corr()
+
+    # Calculating the relation of sample length T to the number of variables N
+    # It's used for de-noising the TIC matrix
+    tn_relation = stock_returns.shape[0] / stock_returns.shape[1]
+
+    # The class that contains the TIC algorithm
+    tic = TIC()
+
+    # Calculating the Theory-Implied Correlation matrix
+    tic_matrix = tic.tic_correlation(tree_classification, corr_matrix, tn_relation, kde_bwidth=0.01)
+
+    # Calculating the distance between the empirical and the theory-implied correlation matrices
+    matrix_distance = tic.corr_dist(corr_matrix, tic_matrix)
+
+Research Notebooks
+==================
```

Comment:
This is the wrong header. Rather use `#################`

Reply:
Changed